core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -355,6 +355,18 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def getBillingProjectSpendConfiguration(billingProjectName: RawlsBillingProjectName): Future[Option[BillingProjectSpendConfiguration]] = {
+    requireProjectAction(billingProjectName, SamBillingProjectActions.alterSpendReportConfiguration) {
```

Comment:
TOL: It seems to me like we should maybe have a `readSpendReportConfiguration` permission?

Reply:
That would definitely be the most granular way of doing it. I decided against it mainly because the two actions are so tightly coupled in how they're used. But they are definitely two distinct actions, so I could be _very_ easily convinced.

I will defer to you to decide if I should do it or not

Reply:
I am re-reading my own comment and it seems like I have good enough reason to make the change and so I will stop being lazy about it and do it :)

Reply:
:D Sounds good! I am happy approving the PR and trusting you to make the chance :D